lib/blockchain_double.js

```diff
@@ -406,6 +407,8 @@ BlockchainDouble.prototype.createBlock = function(parent, emulateParent, callbac
       // Ensure we have the right block number for the VM.
       block.header.number = to.hex(parentNumber + 1);
```

Comment:
We should remove this line for the one below?

Reply:
yes